portfolio/src/main/java/com/google/sps/servlets/MyServletClass.java

```diff
@@ -5,14 +5,30 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.*;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/message")
 public class MyServletClass extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello world, I'm Linda!");
+    String[] bucketListTrips = {"India", "Italy", "Thailand"};
+
+    
+
```

Comment:
Nit (short for nitpick): Please remove these empty lines. You can leave one if it makes it easier to read.

Reply:
Done!  Thanks for the observation!